compiler/src/dotty/tools/dotc/typer/Inliner.scala

```diff
@@ -72,7 +72,8 @@ object Inliner {
   def inlineCall(tree: Tree)(using Context): Tree = {
     val startId = ctx.source.nextId
 
-    if tree.symbol.denot != SymDenotations.NoDenotation && tree.symbol.owner.companionModule == defn.CompiletimeTestingPackageObject
+    if tree.symbol.denot != SymDenotations.NoDenotation
+        && tree.symbol.owner.companionModule == defn.CompiletimeTestingPackageObject then
```

Comment:
Maybe the folowing would be slightly easier to read
```suggestion
       && tree.symbol.owner.companionModule == defn.CompiletimeTestingPackageObject
    then
```

Reply:
yeah that would be more readable, but you merged it already so I don't know what now, create single PR with this single change? ❓

Reply:
will do another PR